src/Core/AdminConsole/Models/Data/IProfileMemberOrganizationDetails.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Bit.Core.AdminConsole.Enums.Provider;
+using Bit.Core.Billing.Enums;
+
+namespace Bit.Core.AdminConsole.Models.Data;
+
+/// <summary>
+/// Interface defining common organization details properties shared between
+/// regular organization users and provider organization users for profile endpoints.
+/// </summary>
+public interface IProfileMemberOrganizationDetails
```

Comment:
It applies to both members and providers, so `IProfileOrganizationDetails` is probably a more accurate name.